src/redux/actions/genes/loadMarkerGenes.js

```diff
@@ -0,0 +1,55 @@
+import {
+  MARKER_GENES_ERROR, MARKER_GENES_LOADING, MARKER_GENES_LOADED,
+} from '../../actionTypes/genes';
+
+import { fetchCachedWork } from '../../../utils/cacheRequest';
+
+const REQUEST_TIMEOUT = 60;
+
+const loadMarkerGenes = (experimentId, resolution) => async (dispatch, getState) => {
+  // Disabled linter because we are using == to check for both null and undefined values
+  // eslint-disable-next-line eqeqeq
+  if (experimentId == null || resolution == null) throw new Error('Null or undefined parameter/s for loadMarkerGenes');
+
+  const { backendStatus, processing } = getState().experimentSettings;
+
+  const { method } = processing.configureEmbedding.clusteringSettings;
+
+  const body = {
+    name: 'MarkerHeatmap',
+    nGenes: 2,
```

Comment:
why 2?

Reply:
it was the number we were using with oliver, just checked with him for what shoudl be teh final value we leave for now and he said 5, so I'll update it in both api hook and ui (here).